lib/Macros/Sources/SwiftMacros/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ add_swift_macro_library(SwiftMacros
   DistributedResolvableMacro.swift
   SyntaxExtensions.swift
   TaskLocalMacro.swift
+  PointerBoundsMacro.swift
```

Comment:
Looks like this file is missing

Reply:
ended up in the wrong directory, moved now